src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -33,7 +34,6 @@
 
 using namespace mlir;
 using namespace mlir::OpTrait::util;
-using namespace mlir::onnxmlir;
```

Comment:
Prefer to explicitly use onnxmlir::StringType to make it clear we are using that "StringType" rather than the "StringType" defined in the krnl dialect.